src/amo/components/PermissionsCard/permissions.js

```diff
@@ -0,0 +1,238 @@
+      clipboardWrite: i18n.gettext('Input data to the clipboard'),
+      devtools: i18n.gettext('Extend developer tools to access your data in open tabs'),
+      downloads: i18n.gettext('Download files and read and modify the browser’s download history'),
+      'downloads.open': i18n.gettext('Open files downloaded to your computer'),
+      find: i18n.gettext('Read the text of all open tabs'),
+      geolocation: i18n.gettext('Access your location'),
+      history: i18n.gettext('Access browsing history'),
+      management: i18n.gettext('Monitor extension usage and manage themes'),
+      // In Firefox the following message replaces the name "Firefox" with the
+      // current brand name, e.g., "Nightly", but we do not need to do that.
+      nativeMessaging: i18n.gettext('Exchange messages with programs other than Firefox'),
+      notifications: i18n.gettext('Display notifications to you'),
+      pkcs11: i18n.gettext('Provide cryptographic authentication services'),
+      proxy: i18n.gettext('Control browser proxy setting'),
+      privacy: i18n.gettext('Read and modify privacy settings'),
+      sessions: i18n.gettext('Access recently closed tabs'),
+      tabs: i18n.gettext('Access browser tabs'),
+      tabHide: i18n.gettext('Hide and show browser tabs'),
+      topSites: i18n.gettext('Access browsing history'),
+      unlimitedStorage: i18n.gettext('Store unlimited amount of client-side data'),
+      webNavigation: i18n.gettext('Access browser activity during navigation'),
+      allUrls: i18n.gettext('Access your data for all websites'),
+      wildcard: (domain) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data for sites in the %(domain)s domain'),
+          { domain }
+        );
+      },
+      // Note that we do not need to use ngettext for this as count will never
+      // be less than 2.
+      tooManyWildcards: (count) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data in %(count)s other domains'),
+          { count: i18n.formatNumber(count) },
+        );
+      },
+      oneSite: (site) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data for %(site)s'),
+          { site }
+        );
+      },
+      // Note that we do not need to use ngettext for this as count will never
+      // be less than 2.
+      tooManySites: (count) => {
+        return i18n.sprintf(
+          i18n.gettext('Access your data on %(count)s other sites'),
+          { count: i18n.formatNumber(count) },
+        );
+      },
+    };
+  }
+
+  // Get a list of permissions from the correct platform file.
+  getCurrentPermissions(
+    addon: AddonType,
+    userAgentInfo: UserAgentInfoType
+  ): Array<string> {
+    const agentOsName =
+      userAgentInfo.os.name && userAgentInfo.os.name.toLowerCase();
+    const platform = userAgentOSToPlatform[agentOsName];
+
+    if (!platform) {
+      log.error(oneLine`No platform exists for user agent OS 
```

Comment:
Hmm, I thought we did too, because my editor (which until now wasn't set up to strip trailing spaces) always bugs me about them. I think eslint ignored this one (both in the linter and in my editor) because it's in a template string. I have now updated my editor to automatically strip extra spaces.